beetmoverscript/tests/test_integration.py

```diff
@@ -232,11 +131,24 @@ def scriptworker_config(tmp_path, config, task):
     return config_path
 
 
-def test_main_android_components_nightly(scriptworker_config, task, boto3_client_mock, aiohttp_session_mock):
+@pytest.mark.parametrize(
+    "config_name, task_name, expected_number_of_artifacts",
+    (
+        ("android-components", "android_components_nightly", 12),
+        ("geckoview", "geckoview_nightly", 16),
+        ("app-services", "application_services_release", 12),
+        ("glean", "glean_release", 20),
+    ),
+)
+def test_main_push_to_maven(config_name, task_name, expected_number_of_artifacts, tmp_path, boto3_client_mock, aiohttp_session_mock):
+    config = get_config(config_name)
+    task = get_test_task(task_name)
+    scriptworker_config = prepare_scriptworker_config(tmp_path, config, task)
```

Comment:
Note to self: `scriptworker_config` is not a fixture anymore because of `get_config(config_name)`

Reply:
Yeah! The need to parametrize the tests (and needing to implement the functionality of `scriptworker_config` for all cases) led to changing the approach like that.